apicast/src/configuration.lua

```diff
@@ -40,10 +42,25 @@ local function regexpify(path)
   return path:gsub('?.*', ''):gsub("{.-}", '([\\w_.-]+)'):gsub("%.", "\\.")
 end
 
+local function check_custom_metric(custom_metric, req)
+  local metric = ngx.req.get_headers()["X-Custom-Metric"]
```

Comment:
IIRC the RFC allows multiple headers with the same name.
That would allow reporting of multiple metrics....

Not supported (i.e. might break?) this code?